core/templates/dev/head/domain/topic_viewer/TopicDataObjectFactory.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Change the year to 2020 only in files you've created in this PR.

Reply:
Oh right, didn't notice it. Thank You @marianazangrossi.